### ISSUE TITLE
Add RSS feed to change log

### ISSUE THREAD

**Issue by Markus00000** [RealFaviconGenerator/realfavicongenerator#165]
It would be great to get notified when a new version is released, especially for releases of major importance. An RSS feed seems appropriate.

**Comment by phbernard** [comment]
Good idea! So far I was asked for a mailing list but I'm not fond of this idea. RSS would be perfect for this task.

**Comment by kevinvanrijn** [comment]
Using GitHub's releases could also work. [You can get an `.atom` file from it](https://github.com/RealFaviconGenerator/realfavicongenerator/releases.atom) or [scrape the page for an `.rss` file](http://createfeed.fivefilters.org/extract.php?url=https%3A%2F%2Fgithub.com%2FRealFaviconGenerator%2Frealfavicongenerator%2Freleases&in_id_or_class=&url_contains=%2Freleases%2Ftag%2F) if you really need one. That way no extra load would be put on the server.

Creating an official RSS feed would probably be more user friendly though.
